wp-content/themes/missilethreat/template-parts/entry-header.php

```diff
@@ -7,28 +7,137 @@
+		
+		<div class="single__header-meta">
+			Last Updated <?php missilethreat_last_updated(); ?>
+		</div>
+
+		<?php  if( 'defsys' === get_post_type() ) { ?>
+			<div class="share-wrapper">
+				<?php missiledefense_system_terms(); ?>
+				<?php missiledefense_share(); ?>
+			</div>
+		<?php } ?>
 
-			the_title( '<h1 class="single__title">', '</h1>' );
+		</div><!-- .entry-header-inner -->
+	<?php } 
 
-			if ( has_excerpt() && is_singular() ) {
-				the_excerpt();
-			}
+	elseif ( 'systems' === get_post_type() || 'actors' === get_post_type() ) {
+		$feat_image = get_the_post_thumbnail_url( $post->ID ); ?>
+		<header class="page__header entry-header"  style="background-image: url('<?php echo $feat_image; ?>')">
+			<div class="overlay"></div>
+			<div class="page__header-inner--narrow">
+				<?php get_template_part( 'template-parts/breadcrumbs' ); ?>
+				<h1 class="page__header-title text--semibold"><?php the_title(); ?></h1>
+				<div class="page__header-divider"></div>
+				<div class="page__header-meta">
+					Last Updated <?php missilethreat_last_updated(); ?>
+				</div>
+			</div>
 
-			missilethreat_authors();
+	<?php }
+	
+	elseif ( is_post_type_archive() ) { 
+    $feat_image = 'style="background-image:url('.get_archive_thumbnail_src( 'missilethreat-fullscreen' ).');"';?>
+		<header class="page__header entry-header" <?php echo $feat_image; ?>>
+			<div class="overlay"></div>
+			<h1 class="page__header-title text--semibold"><?php the_archive_title(); ?></h1>
+			<div class="page__header-divider"></div>
+			<div class="page__header-desc"><?php the_archive_top_content(); ?> </div>
+  
+	<?php } 
+	
+	elseif ( is_archive() ) { ?>
+    <header class="page__header page__header--short entry-header" style='background:linear-gradient(180deg, rgba(7, 52, 74, 0) 0%, rgba(6, 43, 61, 0.9) 75.52%), linear-gradient(90deg, #2A5565 0%, #5F7981 100%);'>
+			<div class="overlay"></div>
+			<h1 class="page__header-title text--semibold"><?php the_archive_title(); ?></h1>
+			<div class="page__header-divider page__header-divider--short"></div>
+			<div class="page__header-desc page__header-desc--short"><?php the_archive_top_content(); ?> </div>
+    
+	<?php } 
+	
+	elseif ( is_page() ) { 
+		$feat_image = get_the_post_thumbnail_url( $post->ID ); ?>
+		<header class="page__header entry-header"  style="background-image: url('<?php echo $feat_image; ?>')">
+			<div class="overlay"></div>
+			<h1 class="page__header-title text--semibold"><?php the_title(); ?></h1>
+			<div class="page__header-divider"></div>
 
-			get_template_part( 'template-parts/featured-image' );
-		?>
+	<?php } 
+	
+	elseif ( is_single() ) { ?>
+		<header <?php post_class('single__header'); ?>>
 
-	</div><!-- .entry-header-inner -->
+			<div class="single__header-wrapper">
+				<div class="single__header-inner">
+					<?php
+					missilethreat_display_categories();
+					the_title( '<h1 class="single__header-title">', '</h1>' );
+					
+					if ( has_excerpt() && is_singular() ) { ?>
```

Comment:
This is duplicative. You already know it's a single page, so check for just having the excerpt.